### DISCUSSION TITLE
What is the difference between AdGuard Home and paid AdGuard client？

### DISCUSSION THREAD

**Discussion by gelald** [AdguardTeam/AdGuardHome#5206] [Q&A] [answered]
Hello! I have been using paid AdGuard client for almost three years. Recently I discovered this free, open source, powerful AdGuard Home! I have some confuses about AdGuard Home and paid AdGuard Client:
1. What is the difference between the two?
2. Can I use both products at the same time?
3. Why do I see that some friends can achieve the function of blocking ads only by using AdGuard Home without using AdGuard client?

Thanks!

**Comment by ghost** [comment, marked as answer]
Very simply, AdGuard Home is a network-wide DNS server designed to block ads and trackers which is generally self-hosted. The most common use here is that you'd install it on a computer on your home network that would be running 24/7, allowing you to block DNS level ads and trackers only. There are however many different ways you can use AdGuard Home - that is just the simplest.

AdGuard AdBlocker is a fully featured allowing for the highest quality of ad and tracker blocking (cosmetic & DNS), but this will only work on supported devices as opposed to a whole network solution.

You can find more information here, as well as many other bits of useful information on all of their products: https://kb.adguard.com/en/general/how-ad-blocking-works

**Reply by gelald** [reply]
A Great Answer👨‍💻

**Reply by ReddyKilowatt** [reply]
What do you mean by "but this will only work on supported devices"?

**Reply by ReddyKilowatt** [reply]
https://adguard.com/en/adguard-home/overview.html

Is also for Adguard home, but a paid version.
How is that different from the open source version?
